vendor.conf

```diff
@@ -0,0 +1,50 @@
+# grpc and protobuf
```

Comment:
How should it look from vndr point of view? `vendor.conf` is not created automatically :/